corptools/src/components/CharAssetGroups.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Panel } from "react-bootstrap";
+import { useQuery } from "react-query";
+import { Bars } from "@agney/react-loading";
+import { loadAssetGroups } from "../apis/Character";
+import { Table } from "react-bootstrap";
+
+const CharAssetGroups = ({ character_id, location_id = 0 }) => {
+  const { isLoading, error, data } = useQuery(
+    ["assetGroups", character_id, location_id],
```

Comment:
It's a decent idea to export your query key names from your API, so that if you need to use the invalidate query functionality you can easily make sure you use the same string.